sci-mathematics/scilab/files/scilab-5.5.2-xmlgraphics-common-2.0.patch

```diff
@@ -0,0 +1,125 @@
+From: Bryan Gardiner <bog@khumba.net>
+Project-Bug-URL: https://bugzilla.scilab.org/show_bug.cgi?id=13724
+Gentoo-Bug-URL: https://github.com/gentoo-science/sci/pull/456
+
+Updates Scilab 5.5.2 to build against xmlgraphics-common >=2.0.  The thirdparty
+changes don't seem to be necessary for Gentoo, but I include them for
+completeness.  This is not tested, as I'm not sure of a test case, and a similar
+patch (where the super.processShape calls are passed cached=true) is already
+awaiting upstream review.
+
+--- a/scilab/etc/classpath.xml.vc
++++ b/scilab/etc/classpath.xml.vc
+@@ -120,7 +120,7 @@ The option is "disableUnderMode" and can be:
+     </path>
+ 
+     <!-- PDF doc generation (dep of jeuclid) -->
+-    <path value="$SCILAB/thirdparty/xmlgraphics-commons-1.4.jar" load="onUse">
++    <path value="$SCILAB/thirdparty/xmlgraphics-commons-2.0.1.jar" load="onUse">
```

Comment:
I think, that the 'thirdparty' references are only for building a bundled Scilab package, and at no point are any of them used on Gentoo.  The thirdparty directory doesn't exist in the built source tree nor in the installed image, and Scilab runs fine without these changes.  So I feel like it would be better to remove these changes rather than to lock the ebuild to a specific version of xmlgraphics-common and batik.  Thoughts?